packages/showcases/svelte/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.svelte

```diff
@@ -57,8 +57,16 @@
     <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
   </VtmnAccordion>
 </Story>
-<Story name="Custom content">
-  <VtmnAccordion summary="Item 1" withIconLeft class="accordion-stories" open>
+<Story name="Custom content" let:args>
+  <VtmnAccordion
+    summary="Item 1"
+    withIconLeft
+    class="accordion-stories"
+    open
+    on:close={() => alert('on close callback')}
+    on:open={() => alert('on open callback')}
```

Comment:
Can you please use the `@storybook/addon-actions`?
Btw, I don't see any `alert` when I've tried to launch your storybook.